src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1396,6 +1438,27 @@ private void OnSpeciesNameTextChanged(string newText)
         editor.NewName = newText;
     }
 
+    /// <summary>
+    ///   "Searches" an organelle selection button by hiding the ones
+    ///   whose name doesn't include the input substring
+    /// </summary>
+    private void OnSearchBoxTextChanged(string newText)
+    {
+        var input = newText.ToLower(CultureInfo.InvariantCulture);
+
+        foreach (VBoxContainer node in organelleSelectionElements)
+        {
+            if (!node.Name.ToLower(CultureInfo.InvariantCulture).Contains(input))
```

Comment:
If Name here is no longer the display name, this can be fixed by getting all organelles from simulation parameters, using their Names to match things and then take the InternalName to map that to the associated Nodes.